types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -1,5 +1,10 @@
+import {ANONYMOUS, SIGNED} from '../../../src/hooks/core/constant';
 import {ChannelList} from '../../database/schema/ChannelList.types';
 
+export interface ContactScreenPayload {
+  from: typeof ANONYMOUS | typeof SIGNED;
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `ContactScreenPayload` interface are appropriate. The `from` property is now typed to accept either `ANONYMOUS` or `SIGNED`, which aligns with the PR's intent of differentiating between anonymous and signed-in users.
```diff
+ export interface ContactScreenPayload {
+   from: typeof ANONYMOUS | typeof SIGNED;
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->